certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+            "description": "The board to get the lanes for.",
+            "x-ms-summary": "Board ID"
+          }
+        ],
+        "description": "Gets all lanes for a specified board."
+      }
+    },
+    "/api/v2/cardTypes": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "type_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "type_id"
+                      },
+                      "icon_type": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "icon_type"
+                      },
+                      "icon_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "icon_id"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "description"
+                      },
+                      "color": {
+                        "type": "string",
+                        "description": "color"
+                      },
+                      "card_color_sync": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "card_color_sync"
+                      },
+                      "all_properties_are_locked": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "all_properties_are_locked"
+                      },
+                      "availability": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "availability"
+                      },
+                      "is_enabled": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "is_enabled"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Types",
+        "operationId": "GetTypes",
+        "parameters": [],
+        "description": "Gets all types within the account."
+      }
+    },
+    "/index.php/api/kanbanize/edit_task/": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@dimitar-kanbanize Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.